developer_guide/installation/setup_hybrid_storage_mysql_mongo.rst

```diff
@@ -15,7 +15,7 @@ Installing and enabling MongoDB support in Akeneo
     :linenos:
 
     $ cd /path/to/installation/pim-community-standard
-    $ php ../composer.phar --prefer-dist require doctrine/mongodb-odm-bundle 3.0.1
+    $ php -d memory_limit=3G ../composer.phar --prefer-dist require doctrine/mongodb-odm-bundle 3.0.1
```

Comment:
Not related to your change but it seems that the `../composer.phar` has an extra dot.